### ISSUE TITLE
Check that quantities to be segmented are correctly passed to DNACopy.

### ISSUE THREAD

**Issue by samuelklee** [broadinstitute/gatk-protected#77]
We use the DNACopy implementation of CBS to segment both targets by log_2 copy ratio and SNPs by absolute shifted alternate allele fraction |alt / (alt + ref) - skew/2|.

In both cases, we take the corresponding quantities x (i.e., log_2 copy ratio or absolute shifted alternate allele fraction) and exponentiate them to get 2^x.  For the targets, this simply gives us the linear copy ratio; it's less clear what the quantity represents for the SNPs.

We then pass 2^x to DNACopy to segment, but there is a parameter DNACopy "data.type" that is set to "logratio," which seems to imply that log ratios should be passed instead.  Since we are passing linear ratios for the targets, if this is true, then the resulting segmentation might be affected; the problem seems to be compounded for the SNPs, but it's hard to compare directly because the absolute shifted alternate allele fraction is not directly proportional to copy ratio.

To sum up, we should double check that things are being segmented correctly, and perhaps also see if there is a better quantity to segment SNPs on.
